vega_sim/api/market.py

```diff
@@ -15,6 +15,7 @@
 • liquidity_monitoring_parameters.auction_extension
```

Comment:
As this is a network parameter experiment, we can remove all these changes to `market.py`